EIPS/eip-7892.md

```diff
@@ -39,12 +41,22 @@ The key motivations for BPO forks are:
 
 ## Specification
 
-BPO forks are a special class of hard fork which **only modifies any of the following** blob-related parameters:
+### Parametric hardforks
+
+BPO forks are a category of **parametric hardforks**, which only change certain protocol parameters at a designated point in time without requiring client-side code changes. The new parameters are activated instantly, with no transition function. These hardforks are consensus-breaking, and therefore receive a `ForkVersion` in the consensus layer.
```

Comment:
This ... isn't a true statement. "Parametric hardforks" are not a thing (yet).

Reply:
Removed this to avoid confusion.